app/package.json

```diff
@@ -1,7 +1,8 @@
 {
-  "name": "openpassport",
+  "name": "@selfxyz/mobile-app",
```

Comment:
need a more descriptive name so it shows up nicely when running commands from root